library/gcse/src/gcse_api.e

```diff
@@ -0,0 +1,243 @@
+note
+	description: "[
+		Simple API to call Google Custome Search Engine
+		Example call:
+		GET https://www.googleapis.com/customsearch/v1?key=INSERT_YOUR_API_KEY&cx=017576662512468239146:omuauf_lfve&q=lectures
+	]"
+	date: "$Date: 2015-10-09 08:11:07 -0300 (vi., 09 oct. 2015) $"
+	revision: "$Revision: 97973 $"
+	EIS: "name=Google Custom Search Engine", "src=https://developers.google.com/custom-search/json-api/v1/using_rest", "protocol=uri"
+
+class
+	GCSE_API
+
+create
+	make
+
+feature {NONE} -- Initialization
+
+	make (a_query_parameters: GCSE_QUERY_PARAMETERS)
+			-- Create an object GCSE with query_parameters `a_query_parameters'
+		do
+			query_paremeter := a_query_parameters
+		ensure
+			query_parameters_set: query_paremeter = a_query_parameters
+		end
+
+feature -- Access
+
+	base_uri: STRING_8 = "https://www.googleapis.com/customsearch/v1"
+			-- Google custom search base URI
+
+	query_paremeter: GCSE_QUERY_PARAMETERS
+			-- Google custom search parameters.
+
+	last_result: detachable GCSE_RESPONSE
+			-- Search results.
+
+feature -- Status Reports
+
+	errors: detachable LIST [READABLE_STRING_8]
+			-- optional table of error codes
```

Comment:
comment does not reflect the attribute.

probably "-- Optional list of error messages."